app/views/services/index.njk

```diff
@@ -5,61 +5,63 @@
 {% endblock %}
 
 {% block mainContent %}
-  {% if new_service_name %}
-    <div class="govuk-grid-column-full govuk-!-margin-0">
-      <div class="flash-container flash-container--good">
-        <p id="new-service-name" class="govuk-body govuk-!-margin-bottom-0">You have been added to {{new_service_name}}</p>
+  <div class="govuk-grid-column-two-thirds">
+    {% if new_service_name %}
+      <div class="govuk-grid-column-full govuk-!-margin-0">
+        <div class="flash-container flash-container--good">
+          <p id="new-service-name" class="govuk-body govuk-!-margin-bottom-0">You have been added to {{new_service_name}}</p>
+        </div>
       </div>
-    </div>
-  {% endif %}
-
-  <h1 class="govuk-heading-l govuk-grid-column-full govuk-!-margin-bottom-3 service-count">
-    {% if services_singular %}
-      You have {{services.length}} service
-    {% else %}
-      You have {{services.length}} services
     {% endif %}
-  </h1>
 
-  <div class="govuk-grid-column-full">
-    <p class="govuk-body">
-      <a href="{{routes.serviceSwitcher.create}}" class="govuk-link govuk-link--no-visited-state create-service">
-        Add a new service
-      </a>
-    </p>
-    {% if has_live_account %}
-    <p class="govuk-body">
-      <a href="{{routes.allServiceTransactions.index}}" class="govuk-link govuk-!-margin-right-3 govuk-link--no-visited-state">
-        View transactions for all live services
-      </a>
-    </p>
-    {% endif %}
-    {% if has_account_with_payouts %}
-    <p class="govuk-body">
-      <a href="{{routes.payouts.list}}" class="govuk-link govuk-!-margin-right-3 govuk-link--no-visited-state">
-        View payments to your bank account
-      </a>
-    </p>
-    {% endif %}
-  </div>
+    <h1 class="govuk-heading-l govuk-grid-column-full govuk-!-margin-bottom-3 service-count">
```

Comment:
can we clean this up to remove all the `govuk-grid-column-full` from all the nested elements, as they're now inside a `govuk-grid-column-two-third` container?

Reply:
Good spot - done.